lib/colorscheme.json

```diff
@@ -9,5 +9,6 @@
   "grey":           { "colorB": "#555" },
   "gray":           { "colorB": "#555" },
   "lightgrey":      { "colorB": "#9f9f9f" },
-  "lightgray":      { "colorB": "#9f9f9f" }
+  "lightgray":      { "colorB": "#9f9f9f" },
+  "purple":         { "colorB": "#6f42c1" }
```

Comment:
Is this github's merged purple? If we do add a named "purple" it probably should not be github's. You can hard-code it in the badge code instead.